tracer/src/Datadog.Trace/Configuration/IntegrationId.cs

```diff
@@ -48,5 +48,6 @@ internal enum IntegrationId
         NLog,
         TraceAnnotations,
         Grpc,
+        ProcessStart
```

Comment:
nit: What do you think of naming this `Process`? It's a little more vague and could allow us to group additional methods into this same "integration" later

Reply:
Good idea!